.github/workflows/actionlint.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@09d2acae674a48949e3602304ab46fd20ae0c42f # v4.0.0
+      - uses: actions/checkout@8edcb1bdb4e267140fa742c62e395cd74f332709 # v4.0.0
```

Comment:
Haha updating this without updating the comment is a bit :yikes:. This appears to be v4.2.2?